markdown/3.x/en/ff-communication.md

```diff
@@ -27,4 +27,17 @@ correctly.
 
 **NOTE**: Although `version` defaults to `7.2` when not set, it is recommended to set it as default value may change with feature releases.
 
-For more information, see the [API reference](/api/3.x/ff-communication#tab=api).
\ No newline at end of file
+For more information, see the [API reference](/api/3.x/ff-communication#tab=api).
+
+### Single hit redirect deprecation
+Due to limitations this functionality brings, it will be removed in future releases.
```

Comment:
```suggestion
Due to the feature's limitations described below, it will be removed in a future version.

```
Mind the empty line. Having this part as a separate paragraph seems reasonable to me.

Reply:
changed